src/pages/_app.tsx

```diff
@@ -1,5 +1,5 @@
-import 'bootstrap/dist/css/bootstrap.min.css'
 import '@src/styles/global.css'
+import '@src/styles/bootstrap.scss'
```

Comment:
Styles should be applied in the order of Bootstrap-> Custom Bootstrap-> our own global.css.  Bootstrap shouldn't overwrite styles in global.css, so I want you to reverse the import order.

Reply:
Fixed.